src/Microsoft.AspNetCore.Antiforgery/AntiforgeryOptions.cs

```diff
@@ -17,10 +17,16 @@ public class AntiforgeryOptions
         private string _formFieldName = AntiforgeryTokenFieldName;
 
         /// <summary>
+        /// The default cookie prefix, which is ".AspNetCore.Antiforgery.".
+        /// </summary>
+        public static readonly string DefaultCookiePrefix = ".AspNetCore.Antiforgery.";
+
+        /// <summary>
         /// Specifies the name of the cookie that is used by the antiforgery system.
         /// </summary>
         /// <remarks>
-        /// If an explicit name is not provided, the system will automatically generate a name.
+        /// If an explicit name is not provided, the system will automatically generate a name 
+        /// which is then appended to the <see cref="DefaultCookiePrefix"/>.
```

Comment:
Hmm, this could be read as if the generated _or an explicit name_ are prefixed by `DefaultCookieToken`. Suggest `... generate a unique name that begins with <see cref="DefaultCookiePrefix"/>.`